gfx/UltimateCataclysm/pngs_normal_32x32/items/spears/spears.json

```diff
@@ -39,6 +39,11 @@
     "fg": "spear_steel",
     "bg": ""
   },
+  {
+    "id": "spear_stone",
+    "fg": "spear_stone",
+    "bg": ""
+  },
```

Comment:
```suggestion
```
You don't need the json entry if the name of the sprite is already the id

Reply:
Thank you for the hint!